charts/harbor/v0.0.1/templates/registry/registry-cm.yaml

```diff
@@ -47,7 +47,7 @@ data:
         {{- if $storage.s3.encrypt }}
         encrypt: {{ $storage.s3.encrypt }}
         {{- end }}
-        {{- if $storage.s3.secure }}
+        {{- if kindIs "bool" $storage.s3.secure }}
```

Comment:
Because `true` is the default value for the `secure` option. 
Originally, if the value is false, the if block is skipped and the false value is not used.
Now, if the value is false, it is bool kind so it returns true

Reply:
This is not very intuitive to understand how to use `$storage.oss.secure`, could you add a new param to enter into this block?

Reply:
Updated